packages/pnpm/src/cmd/help.ts

```diff
@@ -23,39 +23,49 @@ function getHelpText () {
 
         list: [
           {
+            description: 'Install all dependencies for a project.',
```

Comment:
our current convention is no dot when a single sentence is used. And dot if many sentences are used.

Reply:
This has been fixed in the latest commit.